hrm-service/migrations/20200101000000-first_version.js

```diff
@@ -0,0 +1,183 @@
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.transaction(async transaction => {
```

Comment:
At what point do we retire sequelize from these migrations?

Reply:
Now we are technically using umzug directly rather than Sequelize to handle migrations, there's nothing stopping us injecting a pg-promise task rather than a sequelize object to run the SQL logic, it's just a bit of work to convert the old migrations over.

Or we could use the opportunity to have a new baseline schema and get rid of the old migrations, once we're sure all the DBs deployed out there are up to the latest